server/src/internalClusterTest/java/org/opensearch/remotemigration/RemoteStoreMigrationTestCase.java

```diff
@@ -71,4 +86,81 @@ public void testMigrationDirections() {
         updateSettingsRequest.persistentSettings(Settings.builder().put(MIGRATION_DIRECTION_SETTING.getKey(), "random"));
         assertThrows(IllegalArgumentException.class, () -> client().admin().cluster().updateSettings(updateSettingsRequest).actionGet());
     }
+
+    /*
+    Tests end to end remote migration via Blue Green mechanism
+    - Starts docrep nodes with multiple nodes, indices, replicas copies
+    - Adds remote nodes to cluster
+    - Excludes docrep nodes.
+    - Asserts all shards are migrated to remote store
+    - Asserts doc count across all shards
+    - Continuos indexing with refresh/flush happening
+     */
+    public void testEndToEndRemoteMigration() throws Exception {
+        internalCluster().setBootstrapClusterManagerNodeIndex(0);
+        setAddRemote(false);
+        List<String> docRepNodes = internalCluster().startNodes(2);
+        ClusterUpdateSettingsRequest updateSettingsRequest = new ClusterUpdateSettingsRequest();
+        updateSettingsRequest.persistentSettings(Settings.builder().put(REMOTE_STORE_COMPATIBILITY_MODE_SETTING.getKey(), "mixed"));
```

Comment:
Do we need to update this twice? We can use the `initDocRepToRemoteMigration` method in `MigrationBaseTestCase` class directly after all docrep nodes are started

Reply:
done